utils/tzktHooks.ts

```diff
@@ -0,0 +1,152 @@
+import { useEffect, useState } from "react";
+import { TZKT_API_URL } from "../context/config";
+
+/**
+ * Returns the balance of the given address
+ * @param address tz/kt Tezos address
+ */
+export const useTzktBalance = (address: string | null) => {
+  const [balance, setBalance] = useState(0);
+  useEffect(() => {
+    if (address == null) return;
+    (async () => {
+      const response = await fetch(
+        `${TZKT_API_URL}/v1/accounts/${address}/balance`
+      );
+
+      if (response.status == 200) {
+        const json: number = await response.json();
+        setBalance(json / 1_000_000); // Divided by the XTZ decimal
+      }
+    })();
+  }, [address]);
+
+  return balance;
+};
+
+/**
+ * Token type returned by:
+ * https://api.tzkt.io/v1/tokens/balances
+ * Only useful fields are typed
+ */
+type tzkt_token = {
```

Comment:
Type = MAJ :-p

Reply:
fixed in: https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/97ec6759a0b46a923c2fbf948ae9fe2334a5c8bc